tests/admin_views/tests.py

```diff
@@ -4688,6 +4688,23 @@ def test_readonly_backwards_ref(self):
         response = self.client.get(reverse('admin:admin_views_topping_add'))
         self.assertEqual(response.status_code, 200)
 
+    def test_readonly_onetoone_backwards_ref(self):
+        """
+        Regression test for OneToOneField backward reference, when related field is read-only.
+        """
+        v1 = Villain.objects.create(name='Adam')
+        pl = Plot.objects.create(name='Test Plot', team_leader=v1, contact=v1)
+        pd = PlotDetails.objects.create(details='Brand New Plot', plot=pl)
+        response = self.client.get(reverse('admin:admin_views_plotproxy_change', args=(pl.pk,)))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, 'Brand New Plot')
```

Comment:
Actually `assertContains` checks the `status_code` too, so the other assertion is redundant.